packages/block-editor/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## Bug Fixes
+
+-   Fixed a regression that caused the vertical tabs in the inserter (inside the patterns and media sections) to have a nested scrollbar ([#65387](https://github.com/WordPress/gutenberg/pull/65387)).
+
```

Comment:
Well #64371 was only merged two days ago and there’s been no release since right? My apologies if labeling #65837 "regression" was wrong. It would have to actually be released before that label should be applied?

Reply:
Alright fair, I will remove the entry 👍